main.js

```diff
@@ -1,11 +1,23 @@
+// variable declarations to get nodes//
 const menuBox = document.getElementById('menuBox');
 const menuDialog = menuBox.querySelector('#menuDialog');
-const menuBtn = menuDialog.querySelector(".hamburger");
-const closeBtn = menuDialog.querySelector(".times");
-const menuLinks = menuDialog.querySelector(".nav-links")
+const menuBtn = menuDialog.querySelector('.hamburger');
+const closeBtn = menuDialog.querySelector('.times');
+const menuLinks = menuDialog.querySelector('.nav-links');
 
-menuBtn.addEventListener('click', displayMenu);
+// displayMenu function declaration //
+function displayMenu() {
+  menuBox.classList.toggle('menuDialog');
+}
 
-closeBtn.addEventListener('click', closeMenu);
+// closeMenu function declaration //
+function closeMenu() {
```

Comment:
- [ ] Kindly fix the horizontal overflow on the page, so the close button on the mobile menu so the close button can be seen without scrolling left and your design look like the Figma design

|Your design|Figma design|
|-|-|
|![Screenshot_20220914_172004](https://user-images.githubusercontent.com/25227022/190209924-acb24b5f-d378-43de-a01d-f6a457e50782.png)|![Screenshot_20220914_172212](https://user-images.githubusercontent.com/25227022/190209925-74850441-88eb-47ba-a169-1af8dd2f7c75.png)|